src/campsite-info/portage-info.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FunctionComponent } from 'react';
+import { CampHeader } from './components/camp-header';
+import { CampImages } from './components/camp-images';
+import { Box } from '@mui/material';
+import { graphicsStyles, mapStyles, imageStyles } from './campsite-info.styles'
+
+export const CampsiteInfo: FunctionComponent = () => {
+    return (
+        <div>
```

Comment:
whys is everything in here in a div? Is it just because React won't let you return a bunch of sibling elements from a function component?